src/main_benchmark.cpp

```diff
@@ -419,32 +419,47 @@ int main(int argc, char** argv) {
         return EX_USAGE;
     }
     // input names handling
-    std::string cliInputsNames(cliparser.result->operator[]("inputs_names").as<std::string>());
-    auto inputsNames = ovms::tokenize(cliInputsNames, ',');
-    if (inputsNames.size() != 1) {
-        std::cout << __LINE__ << std::endl;
-        return EX_USAGE;
+    OVMS_ServableMetadata* metadata;
+    OVMS_GetServableMetadata(srv, servableName.c_str(), servableVersion, &metadata);
```

Comment:
It's good idea to not use user provided input names and use C-API here but if we are using Metadata then it doesn't make sense to me to maintain both paths.